ledger-service/db-backend/BUILD.bazel

```diff
@@ -34,6 +34,7 @@ da_scala_library(
         "@maven//:org_scalaz_scalaz_core_2_12",
         "@maven//:org_tpolecat_doobie_core_2_12",
         "@maven//:org_tpolecat_doobie_free_2_12",
+        "@maven//:org_tpolecat_doobie_postgres_2_12",
```

Comment:
```suggestion
```

Use the same technique used for `insertContracts`: take `Get[Array[String]]` as an implicit argument to `selectContracts`.